Makefile

```diff
@@ -70,6 +70,14 @@ post-install:
 	@cp ./native_modules/ImagePickerModule.java node_modules/react-native-image-picker/android/src/main/java/com/imagepicker
 	@# Need to copy custom RNDocumentPicker.m that implements direct access to the document picker in iOS
 	@cp ./native_modules/RNDocumentPicker.m node_modules/react-native-document-picker/ios/RNDocumentPicker/RNDocumentPicker.m
+	@# Add caching support for native module constants
+	@rm node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java
+	@rm node_modules/react-native-fetch-blob/android/src/main/java/com/RNFetchBlob/RNFetchBlob.java
+	@rm node_modules/react-native-navigation/android/app/src/main/java/com/reactnativenavigation/NavigationApplication.java
+	@cp ./native_modules/RNDeviceModule.java node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo
+	@cp ./native_modules/RNFetchBlob.java node_modules/react-native-fetch-blob/android/src/main/java/com/RNFetchBlob
+	@cp ./native_modules/NavigationApplication.java node_modules/react-native-navigation/android/app/src/main/java/com/reactnativenavigation
```

Comment:
are there any other changes than the telemetry stuff? cause if that's the case I don't think it should be part of the PR